share/locale/languages.json

```diff
@@ -1,72 +1,216 @@
+    },
+    "be": {
+        "name": "Беларуская мова"
+    },
+    "bg": {
+        "name": "български"
+    },
+    "br": {
+        "name": "Brezhoneg"
+    },
+    "ca": {
+        "name": "Català"
+    },
+    "ca@valencia": {
+        "name": "Valencià"
+    },
+    "cs": {
+        "name": "Čeština"
+    },
+    "cy": {
+        "name": "Cymraeg"
+    },
+    "da": {
+        "name": "Dansk"
+    },
+    "de": {
+        "name": "Deutsch"
+    },
+    "el": {
+        "name": "ελληνικά"
+    },
+    "en_GB": {
+        "name": "English (GB)",
+        "fallbackLanguages": ["en_US"]
+    },
+    "en_US": {
+        "name": "English (US)"
+    },
+    "eo": {
+        "name": "Esperanto"
+    },
+    "es": {
+        "name": "Español"
+    },
+    "et": {
+        "name": "Eesti"
+    },
+    "eu": {
+        "name": "Euskara"
+    },
+    "fa": {
+        "name": "فارس"
+    },
+    "fi": {
+        "name": "Suomi"
+    },
+    "fil": {
+        "name": "Filipino"
+    },
+    "fo": {
+        "name": "Føroyskt"
+    },
+    "fr": {
+        "name": "Français"
+    },
+    "ga": {
+        "name": "Gaeilge"
+    },
+    "gd": {
+        "name": "Gàidhlig"
+    },
+    "gl": {
+        "name": "Galego"
+    },
+    "he": {
+        "name": "עברית"
+    },
+    "hi_IN": {
+        "name": "हिन्दी"
+    },
+    "hr": {
+        "name": "Hrvatski"
+    },
+    "hu": {
+        "name": "Magyar"
+    },
+    "hy": {
+        "name": "Հայերեն"
+    },
+    "id": {
+        "name": "Bahasa Indonesia"
+    },
+    "ig": {
+        "name": "Ásụ̀sụ̀ Ị̀gbò"
+    },
+    "it": {
+        "name": "Italiano"
+    },
+    "ja": {
+        "name": "日本語"
+    },
+    "ka": {
+        "name": "ქართული"
+    },
+    "kab": {
+        "name": "ⵜⴰⵇⴱⴰⵢⵍⵉⵜ"
+    },
+    "ko": {
+        "name": "한국어"
+    },
+    "lt": {
+        "name": "Lietuvių"
+    },
+    "lv": {
+        "name": "Latviešu"
+    },
+    "ml": {
+        "name": "മലയാളം"
+    },
+    "mn_MN": {
+        "name": "Монгол хэл"
+    },
+    "mt": {
+        "name": "Malti"
+    },
+    "nb": {
+        "name": "Norsk (Bokmål)"
+    },
+    "nl": {
+        "name": "Nederlands"
+    },
+    "nn": {
+        "name": "Norsk (Nynorsk)"
+    },
+    "pl": {
+        "name": "Polski"
+    },
+    "pt": {
+        "name": "Português"
+    },
+    "pt_BR": {
+        "name": "Português brasileiro"
+    },
+    "ro": {
+        "name": "Română"
+    },
+    "ru": {
+        "name": "Русский"
+    },
+    "scn": {
+        "name": "Sicilianu"
+    },
+    "sk": {
+        "name": "Slovenčina"
+    },
+    "sl": {
+        "name": "Slovenščina"
+    },
+    "sr": {
+        "name": "Srpski"
+    },
+    "sr_RS": {
+        "name": "Српски"
+    },
+    "sv": {
+        "name": "Svenska (modern)"
+    },
+    "sv_SE": {
+        "name": "Svenska (traditionell)"
+    },
+    "th": {
+        "name": "ภาษาไทย"
+    },
+    "tr": {
+        "name": "Türkçe"
+    },
+    "uk": {
+        "name": "Українська"
```

Comment:
might fall back to "Polski", I've been told by Polish and Ukrainians that they share about 80% of the language.
But Cyrillic rather than Latin letters...

Reply:
If anything ukrainian should fall back to russian, but certainly not polish

Reply:
Certainly not Russian, the the overlap is at most 60%, and politically this won't be wanted too

Reply:
Polish might not ba a good idea either... too far fetched

Reply:
Here in Ukraine, most people understand Russian more than Polish. So Russian is fine for a fallback. 
Polish will look strange because it has a Latin alphabet

Reply:
Of course in the end the users and translators would have to decide and come to an agreement.